### PR TITLE
Expose and document the spawn source command

### PR DESCRIPTION
**Pull request by dominiklohmann**

### CODE REVIEW

File: doc/cli/vast-spawn-source-csv.md [hunk 1]
```diff
@@ -0,0 +1,7 @@
+The `spawn source csv` command spawns a CSV source inside the node and is the
+analog to the `import csv` command.
+
+For more information, please refer to the [documentation for the `spawn
+source`](https://docs.tenzir.com/vast/cli/vast/spawn/source) and [documentation
+for the `import csv`](https://docs.tenzir.com/vast/cli/vast/import#import-csv)
+commands.
```
Comment (mavam) [inline doc/cli/vast-spawn-source-csv.md:7]:
```suggestion
For more information, please refer to the documentation for [`spawn
source`](https://docs.tenzir.com/vast/cli/vast/spawn/source) and
[`import csv`](https://docs.tenzir.com/vast/cli/vast/import#import-csv).
```

Also applies to the other file.

**Comment by 0ortmann** [comment]
Is the content in `cli` autogenerated.

**Comment by dominiklohmann** [comment]
> Is the content in `cli` autogenerated.

If you mean on the docs website: yes.

File: doc/cli/vast-spawn-source-csv.md (hunk 1 above)
Comment (dominiklohmann) [reply doc/cli/vast-spawn-source-csv.md:7]: Done.

**Comment by mavam** [comment]
Exposing spawn only makes sense if we add the corresponding shutdown command. You can do that in here or via a separate PR.

**Comment by dominiklohmann** [comment]
> Exposing spawn only makes sense if we add the corresponding shutdown command. You can do that in here or via a separate PR.

We do have a ticket for that, which I'll schedule for next week. In order to do that nicely, we must expose the label of the spawned component on the command line. The kill command is barely usable otherwise.

**Pull request merged**